.github/workflows/push-master.yml

```diff
@@ -17,21 +17,15 @@ on:
   push:
     branches:
       - documentation-test-branch
```

Comment:
```suggestion
      - master
```

Reply:
We still need to make this change. Things are correctly rebuilt but we want it to be for `master`.